src/views/tutorial-library/components/results/index.tsx

```diff
@@ -24,17 +36,30 @@ export function TutorialLibraryResults() {
 		)
 	}
 
+	let itemsToRender = null
+	if (hasFiltersApplied) {
+		itemsToRender = hits.map((hit) => (
+			<TutorialCardWithBookmark
+				key={hit.objectID}
+				{...getTutorialCardPropsFromHit(hit)}
+			/>
+		))
+	} else {
+		itemsToRender = defaultTutorials.map((tutorial) => (
+			<TutorialCardWithBookmark
+				key={tutorial.id}
+				{...formatTutorialCard({
+					...tutorial,
+					defaultContext: tutorial.collectionCtx.default,
+				})}
+			/>
+		))
+	}
```

Comment:
A future enhancement might be to somehow fetch the initial data from algolia as well so we don't have to handle the two different formats 💭

Reply:
💯 It could be cool for the default grid to be like 'most popular' tutorials or something.